source/Halibut.Tests/LowerHalibutLimitsForAllTests.cs

```diff
@@ -21,8 +23,8 @@ public static void LowerHalibutLimits()
             halibutLimitType.ReflectionSetFieldValue(nameof(HalibutLimits.ConnectionErrorRetryTimeout), TimeSpan.FromSeconds(66)); // Must always be greater than the heartbeat timeout.
             
             // Intentionally set higher than the heart beat, since some tests need to determine that the hart beat timeout applies.
-            halibutLimitType.ReflectionSetFieldValue(nameof(HalibutLimits.TcpClientSendTimeout), TimeSpan.FromSeconds(45));
-            halibutLimitType.ReflectionSetFieldValue(nameof(HalibutLimits.TcpClientReceiveTimeout), TimeSpan.FromSeconds(45));
+            halibutLimitType.ReflectionSetFieldValue(nameof(HalibutLimits.TcpClientSendTimeout), HalftheTcpRecieveTimeout + HalftheTcpRecieveTimeout);
```

Comment:
I think net48 doesn't have `/` or `*` operators for `TimeSpan` so defining the full values as the sum of its half should work and always be correct.